_studio/shared/include/mfx_config.h

```diff
@@ -50,25 +50,21 @@
 
 #if !defined(LINUX_TARGET_PLATFORM) || defined(LINUX_TARGET_PLATFORM_BDW) || defined(LINUX_TARGET_PLATFORM_CFL) || defined(LINUX_TARGET_PLATFORM_BXT)
     #if !defined(ANDROID)
-        // h264d
+        // HW decoders are part of library
         #define MFX_ENABLE_H264_VIDEO_DECODE
         #define MFX_ENABLE_H265_VIDEO_DECODE
         #define MFX_ENABLE_VP8_VIDEO_DECODE_HW
-        //#define MFX_ENABLE_VP9_VIDEO_DECODE_HW
-
-        // h265d
-        #if defined(AS_HEVCD_PLUGIN) || defined(AS_HEVCE_PLUGIN)
-            #define MFX_ENABLE_H265_VIDEO_DECODE
-        #endif
-
+        #define MFX_ENABLE_VP9_VIDEO_DECODE_HW
+        
         //h264e
         #define MFX_ENABLE_H264_VIDEO_ENCODE
 
         //h265e
-        #if defined(AS_HEVCD_PLUGIN) || defined(AS_HEVCE_PLUGIN) || defined(MFX_VA)
+        #if defined(AS_HEVCE_PLUGIN) || defined(MFX_VA)
+            #define MFX_ENABLE_H265_VIDEO_DECODE
```

Comment:
This is inconsistent with `#if defined(AS_HEVCE_PLUGIN)`

Reply:
Will fix.